.yarnrc.yml

```diff
@@ -18,6 +18,15 @@ packageExtensions:
   "@swc-node/register@^1.9.0":
     dependencies:
       "@swc/types": "*"
+  "@typescript-eslint/rule-tester@*":
+    dependencies:
+      "@typescript-eslint/parser": "~8.12.2"
+  "angular-eslint@*":
+    dependencies:
+      "@typescript-eslint/utils": ~8.12.2
+  "@angular-eslint/eslint-plugin-template@*":
+    dependencies:
+      "@typescript-eslint/types": "^8.0.0"
```

Comment:
This dep is declared as peerDep but missing in `angular-eslint`
I tried the following code but it still complains, that's the only way I found to fix it
```yml
"angular-eslint@*":
    dependencies:
      "@typescript-eslint/utils": ~8.12.2      
      "@typescript-eslint/types": "^8.0.0"